### DISCUSSION TITLE
v0.6.1 🌈

### DISCUSSION THREAD

**Discussion by moqsien** [gvcgo/version-manager#48] [General]
## What’s Changed

* 全新的TUI，交互式，更少的命令，更少的用户输入，更直观的提示 @moqsien
* 全新的版本仓库，[vsources](https://github.com/gvcgo/vsources) @moqsien
* 全新的Installer设计，保证可扩展性，通过使用**conda**，甚至可以扩展到支持数以千计的应用 @moqsien
* 支持了更多语言，例如erlang, elixir, ruby, clang, gcc, lfortran, r, perl, lua等 @moqsien
* 增加了windows可执行文件签名，减少windows defender误报病毒 @moqsien
* 新的shell，修复了相关问题，支持了fish shell @zzhaolei @moqsien 
* 拆分 cmd.go 中的子命令到不同的文件 (#39) @zzhaolei @moqsien 
* 修复命令行自动补全问题 @zzhaolei 

## 👻 Maintenance

* Bump golang.org/x/net from 0.19.0 to 0.23.0 in the go_modules group across 1 directory (#44) @dependabot

<hr /><em>This discussion was created from the release <a href='https://github.com/gvcgo/version-manager/releases/tag/v0.6.1'>v0.6.1 🌈</a>.</em>

**Comment by cbyFlyPlan** [comment]
Get "https://raw.githubusercontent.com/gvcgo/vsources/main/sdk-list.version.json": unexpected EOF 为什么会出现这个错？
